### PR TITLE
Update whos using to more examples

### PR DESCRIPTION
**Pull request by motdotla**
@maxbeatty and @jcblw - anyone come to mind that is using dotenv? I'd like to update this to a handful (4-5) more popular/well-known projects.

### CODE REVIEW

**Comment by jcblw** [comment]
was just think about this last night when I sent you that jaws repo. I'll comb through the [dependents list](https://www.npmjs.com/browse/depended/dotenv) see if I can find any other good ones.

**Comment by jcblw** [comment]
added `node-lambda` and `resume-cli`

was think about adding

* [phant](https://www.npmjs.com/package/phant)
* [adafruit-io-node](https://github.com/adafruit/adafruit-io-node)

**Comment by motdotla** [comment]
:+1: and :+1: on adding phant and adafruit-io-node. It shows use in 'hardware/things'

**Comment by ahmadnassri** [comment]
use it in all my projects (OSS or otherwise) some highlighted ones:

- [Mockbin](https://www.npmjs.com/package/mockbin)
- [echint](https://www.npmjs.com/package/echint)

and more coming ... 

:)

**Comment by ahmadnassri** [comment]
also in use on all [Mashape](https://www.Mashape.com) products. *(if you wanted corporate examples)*

**Comment by jcblw** [comment]
@ahmadnassri great added Mockbin to the list @motdotla think this is a pretty good list @maxbeatty any you wanna add

**Comment by motdotla** [comment]
This looks great :+1: :heart: the addition of Mockbin. (thanks for that @ahmadnassri) and nice link to 'many more' @jcblw.

File: README.md [hunk 1]
```diff
@@ -203,8 +203,10 @@ See [Contributing Guide](Contributing.md)
...
-* [google-oauth2-service-account](https://github.com/jacoblwe20/google-oauth2-service-account)
-* [kibble](https://github.com/motdotla/kibble)
-* [github-streaker](https://github.com/motdotla/github-streaker)
+* [jaws](https://github.com/jaws-framework/jaws-core-js)
+* [node-lambda](https://github.com/motdotla/node-lambda)
+* [resume-cli](https://www.npmjs.com/package/resume-cli)
+* [phant](https://www.npmjs.com/package/phant)
+* [adafruit-io-node](https://github.com/adafruit/adafruit-io-node)
+* [mockbin](https://www.npmjs.com/package/mockbin)
+* [and many more...](https://www.npmjs.com/browse/depended/dotenv)
```
Comment (maxbeatty) [inline README.md]: pretty swell of npm to offer such a nice link :heart_eyes_cat:

**Comment by bjoshuanoah** [comment]
My "require-environment-variables" module should be on the list: https://www.npmjs.com/package/require-environment-variables

**Comment by motdotla** [comment]
@bjoshuanoah :heart: this approach - we had need for this @Rebelmail and used convict at the time. This is more straightforward. I'm going to create a separate piece to show off things that 'Go well with dotenv'.  I just came across another here from @rolodato

**Comment by bjoshuanoah** [comment]
@jcblw contributed as well.

**Pull request merged**